suite-native/module-send/src/components/SendOutputsForm.tsx

```diff
@@ -0,0 +1,149 @@
+const constructFormDraft = ({ outputs }: SendOutputsFormValues): FormState => ({
+    outputs:
+        outputs?.map(({ address, amount }) => ({
+            type: 'payment',
+            address,
+            amount,
+            token: null,
+            fiat: '0',
+            currency: { label: 'usd', value: '1000' },
+        })) ?? [],
+    isCoinControlEnabled: false,
+    hasCoinControlBeenOpened: false,
+    selectedUtxos: [],
+    feeLimit: '0',
+    feePerUnit: '0',
+    options: [],
+});
+
+export const SendOutputsForm = ({ accountKey }: SendFormProps) => {
+    const dispatch = useDispatch();
+    const debounce = useAsyncDebounce();
+    const { showToast } = useToast();
+    const navigation =
+        useNavigation<StackNavigationProps<SendStackParamList, SendStackRoutes.SendReview>>();
+
+    const account = useSelector((state: AccountsRootState) =>
+        selectAccountByKey(state, accountKey),
+    );
+    const networkFeeInfo = useSelector((state: FeesRootState) =>
+        selectNetworkFeeInfo(state, account?.symbol),
+    );
+    const sendFormDraft = useSelector((state: SendRootState) =>
+        selectSendFormDraftByAccountKey(state, accountKey),
+    );
+
+    const form = useForm<SendOutputsFormValues>({
+        validation: sendOutputsFormValidationSchema,
+        context: {
+            networkFeeInfo,
+            networkSymbol: account?.symbol,
+            availableAccountBalance: account?.availableBalance,
+        },
+        defaultValues: {
+            outputs: [
+                {
+                    address: 'bcrt1q7r9yvcdgcl6wmtta58yxf29a8kc96jkyyk8fsw',
+                    amount: sendFormDraft?.outputs[0]?.amount,
+                },
+            ],
+        },
+    });
+
+    const {
+        handleSubmit,
+        control,
+        getValues,
+        formState: { isValid, isSubmitting },
+    } = form;
+    const watchedFormValues = useWatch({ control });
+
+    const storeFormDraftIfValid = useCallback(async () => {
+        await dispatch(
+            sendFormActions.storeDraft({
+                accountKey,
+                formState: constructFormDraft(getValues()),
+            }),
+        );
+    }, [accountKey, dispatch, getValues]);
+
+    useEffect(() => {
+        if (watchedFormValues && isValid) debounce(storeFormDraftIfValid);
+    }, [isValid, storeFormDraftIfValid, watchedFormValues, debounce]);
+
+    const handleNavigateToReviewScreen = handleSubmit(async () => {
+        // Ignore another submit button clicks until is the submit not finished.
+        if (isSubmitting) return;
+
+        // TODO: navigate to SendFeeScreen instead, when ready
+        // issue: https://github.com/trezor/trezor-suite/issues/10871
+        navigation.navigate(SendStackRoutes.SendReview, { accountKey });
+        const response = await dispatch(
+            onDeviceTransactionReviewThunk({
+                accountKey,
+                formState: constructFormDraft(getValues()),
```

Comment:
You don't need to `getValues()`, `handleSubmit` passes these as a parameter to the anonymous function so they can be used here.

Reply:
good point, fixed 3fd9ef1399